### DISCUSSION TITLE
Jump to the next epoch without finish this epoch

### DISCUSSION THREAD

**Discussion by SilenceZZZZZZ** [Lightning-AI/pytorch-lightning#12380] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Epoch 0:   1%|          | 96/15051 [00:21<55:05,  4.52it/s, loss=4.57, v_num=1]
Epoch 0:   0%|          | 0/15051 [00:00<?, ?it/s, loss=4.57, v_num=1]         
Epoch 1:   0%|          | 0/15051 [00:00<?, ?it/s, loss=4.57, v_num=1]
Epoch 1:   0%|          | 1/15051 [00:03<15:55:07,  3.81s/it, loss=4.57, v_num=1]

Dose anyone know why?

**Comment by rohitgr7** [comment]
hey @SilenceZZZZZZ !

this shouldn't happen in an ideal case. Did you see any logs/warnings?

if not, is it possible to share a reproducible script using: https://colab.research.google.com/github/PyTorchLightning/pytorch-lightning/blob/master/pl_examples/bug_report/bug_report_model.ipynb
